database/migrations/2019_03_13_000000_create_page_manager_tables.php

```diff
@@ -20,7 +20,7 @@ public function up()
             $table->timestamps();
             $table->enum('type', ['page', 'region']);
             $table->string('name');
-            $table->string('slug')->default('')->unique();
```

Comment:
this line creates a unique key containing the table name.  default config would result in a key named nova_page_manager_slug_unique.  if you changed the table name to "pages", the key would be pages_slug_unique.

Reply:
When this is dropped later, it doesn't exist with a custom table name.